android/app/src/main/java/com/epfl/dedis/hbt/ui/wallet/TransactionState.kt

```diff
@@ -0,0 +1,20 @@
+package com.epfl.dedis.hbt.ui.wallet
+
+import com.epfl.dedis.hbt.data.model.PendingTransaction
+import com.epfl.dedis.hbt.data.model.Transaction
+
+/**
+ * Describe the five states of a transaction process :
```

Comment:
Agree. Just one comment here: this will need to be extended in the future with the "voucher" use case.